libraries/rush-lib/assets/rush-init/rush.json

```diff
@@ -4,7 +4,6 @@
  */
 {
   "$schema": "https://developer.microsoft.com/json-schemas/rush/v5/rush.schema.json",
-
```

Comment:
Looks like your Prettier extension messed up the formatting in this file.  Is it for some reason ignoring the config:

https://github.com/microsoft/rushstack/blob/b0bd583db06237e9fac48dc70ddf4f9c0c444e04/.prettierignore#L101-L102

Reply:
Fixed